agent/src/config.cc

```diff
@@ -143,8 +143,8 @@ config::config(const std::string& path) {
                   ? to_file
                   : to_stdout;
   _log_file = json_config.get_string("log_file", "");
-  _log_files_max_size = json_config.get_unsigned("log_files_max_size", 0);
-  _log_files_max_number = json_config.get_unsigned("log_files_max_number", 0);
+  _log_files_max_size = json_config.get_unsigned("log_max_file_size", 0);
+  _log_files_max_number = json_config.get_unsigned("log_max_files", 0);
```

Comment:
isn't it possible to keep the same names?
storing "log_max_file_size" in _log_max_file_size...
"log_max_files" in _log_max_files in log_max_files ?